microsoft-365/knowledge/topic-experiences-discovery-curation.md

```diff
@@ -37,40 +37,16 @@ Additionally, users will also be able to find topics through Microsoft Search.
 
 ## Topic curation
 
-Topic Experiences welcomes human "curation" to improve the quality of your topics. While AI initially identifies and suggests topics, manually made updates to content from contributors, confirmation from users for AI generated content, and feedback on the usefulness of topics are all essential.
+Topic Experiences welcomes human contribution to improve the quality of your topics. While AI initially identifies and suggests topics, manually made edits to content from contributors, confirmation from users for AI generated content, and feedback on the usefulness of topics are all essential.
 
 - AI generated topics ("suggested topics") can be reviewed by **knowledge managers** in your organization. In the Manage Topics page in the Topic Center, they can choose to confirm them as valid, or reject them to prevent them from being viewed.
 
 - You can assign *Create and edit topics* permissions to any of your licensed users so that they can make changes to existing topics or create new topics when needed. 
 
-- Even users who only have read access to topic (topic viewers) will be asked to verify the usefulness of specific topics.
-
-Even with human curation, AI will continually look for more information about topics, and will look for human verification. For example, if AI thinks you are a person that should be pinned as an expert on a topic, it will ask you to confirm this. 
-
-
-
-
-
-
-
-
-
-
-
-
-
+- Even users who only have read access to topic (topic viewers) will be asked to verify the usefulness of specific topics. Their feedback is also taken to confirm or reject a suggested topic.
 
+Even with human edits, AI will continually look for more information about topics, and will look for human verification. For example, if AI thinks you are a person that should be listed as an expert on a topic, it will ask you to confirm this. 
```

Comment:
```suggestion
Even with human edits, AI will continually look for more information about topics and will look for human verification. For example, if AI thinks you are a person that should be listed as an expert on a topic, it will ask you to confirm this. 
```